src/clients/QBittorrent.ts

```diff
@@ -224,6 +225,14 @@ export default class QBittorrent implements TorrentClient {
 		autoTMM: boolean;
 		category: string;
 	}> {
+		const { dataDirs } = getRuntimeConfig();
+		if (dataDirs.length > 0) {
+			const save_path: string = join(searchee.path, "/..");
```

Comment:
does `path.dirname(searchee.path)` work here instead of `path.join(searchee.path, '/..')`?